test/tests/landing-page.js

```diff
@@ -39,20 +39,17 @@ describe('beaker landing page', function() {
   });
 
   it('has a file menu with 4 items', function () {
-    beakerPO.mainmenu.get(0).click();
-    beakerPO.sync();
+    beakerPO.mainmenu.get(0).element(by.css('.dropdown-toggle')).click();
```

Comment:
can you add a function to the page object for menu access so we can abstract `by.css('.dropdown-toggle')` out of the code?